scripts/tasks_retrival/Reactome_tasks_creation.py

```diff
@@ -90,33 +111,43 @@ def dump_to_task(task_dir, outcomes_df):
     "--pathways-file",
     type=click.STRING,
     help="Path to the pathways files from reactome available using the analysis GUI",
-    default="",
+    default=None,
 )
 @click.option(
     "--top-pathways-file",
     type=click.STRING,
     help="The location of the ReactomePathwaysRelation file available at https://reactome.org/download-data",
-    default="",
+    default=None,
 )
 def main(
     main_task_directory, task_name, allow_downloads, pathways_file, top_pathways_file
 ):
+
     if allow_downloads:
-        symb_list = get_symbol_list()
-        token = get_token(symb_list)
-        url = f"https://reactome.org/AnalysisService/download/{token}/pathways/TOTAL/result.csv"
-        df_path = pd.read_csv(url, index_col="Pathway identifier")
-        top_level = get_top_level_pathway()
-    else:
-        df_path = pd.read_csv(pathways_file)
-        top_level = pd.read_csv(top_pathways_file)
+        reactom_url = (
+            get_token_link_for_symbols(get_symbol_list()) if allow_downloads else ""
+        )
+
+    pathways_file = verify_source_of_data(
+        pathways_file, url=reactom_url, allow_downloads=allow_downloads
+    )
+    top_pathways_file = verify_source_of_data(
+        top_pathways_file, url=TOP_PATHWAYS_URL, allow_downloads=allow_downloads
+    )
+    df_path = pd.read_csv(pathways_file, index_col="Pathway identifier")
+
```

Comment:
should `index_col` be i the header ?

Reply:
I moved this into an external method. and now it is in the header